airbyte-integrations/connectors/destination-azure-blob-storage/src/main/resources/spec.json

```diff
@@ -35,13 +35,23 @@
         "examples": ["airbyte5storage"]
       },
       "azure_blob_storage_account_key": {
+        "title": "Azure Blob Storage account key",
         "description": "The Azure blob storage account key.",
         "airbyte_secret": true,
         "type": "string",
         "examples": [
           "Z8ZkZpteggFx394vm+PJHnGTvdRncaYS+JhLKdj789YNmD+iyGTnG+PV+POiuYNhBg/ACS+LKjd%4FG3FHGN12Nd=="
         ]
       },
+      "azure_blob_storage_output_buffer_size": {
+        "title": "Azure Blob Storage output buffer size",
```

Comment:
A little of both, the Azure SDK automatically sets the block size. So if you set the buffer too small you can have the issue that prompted this issue (50,000 block limit). Setting larger buffered chunks should maximize the block size to avoid block count issues and improve performance, in theory at least. However, going too high could cause issues on the system as the streams don't close until the full sync job, or job, is completed.

I do like the idea of the connector handling it, though I think it could get tricky trying to do it there.